app/templates/emails/brief_clarification_question.html

```diff
@@ -27,6 +27,6 @@
   Thanks,<br>
```

Comment:
Is there a rogue opening `<p>` tag on line 26?  Or a missing `</p>`?  This might be in the other templates.